civiclens/collect/upload_bulk_csvs_to_db.py

```diff
@@ -0,0 +1,178 @@
+from datetime import datetime
+import polars as pl
+import argparse
+
+from civiclens.collect.access_api_data import pull_reg_gov_data
+from civiclens.collect.move_data_from_api_to_database import (
+    insert_comment_into_db,
+)
+from civiclens.utils.constants import REG_GOV_API_KEY
+
+"""
+This code is for one-time and ad hoc bulk uploads of comments.
+Csv files are obtained for known documents here: https://www.regulations.gov/bulkdownload
+Adding comments to the db is faster with this method than via API 
+"""
+
+
+def load_data(file_name: str) -> pl.DataFrame([]):
+    """
+    Loads csv file into a polars df of comments and filters out null comments
+    """
+    q = pl.scan_csv(file_name).filter(~pl.col("Comment").is_null())
+    df = q.collect()
+
+    return df
+
+
+def get_document_objectId(doc_id: str) -> str:
+    """
+    Fetches the document objectId from the regulations.gov API
+
+    Input: doc_id (str): the id of a document (the id you search with on regulations.gov)
+
+    Returns: objectId (str): the objectId, which we use to link database tables
+    """
+    doc_data = pull_reg_gov_data(
+        REG_GOV_API_KEY, "documents", params={"filter[searchTerm]": doc_id}
+    )
+
+    return doc_data[0]["attributes"]["objectId"]
+
+
+def format_date(datetime_str: str) -> str:
+    """
+    Format a datetime string to the desired ISO 8601 format with UTC ('Z' timezone)
+    """
+    if not datetime_str:
+        return ""
+
+    try:
+        # Parse the input datetime string (assuming it's in ISO 8601 format)
+        # Use datetime.strptime with the correct format
+        parsed_datetime_dt = datetime.strptime(datetime_str, "%Y-%m-%dT%H:%MZ")
+
+        # Convert the parsed datetime object to the desired format string
+        formatted_datetime_str = parsed_datetime_dt.strftime("%Y-%m-%dT%H:%M:%SZ")
+        return formatted_datetime_str
+    except Exception as e:
+        print(f"Error parsing datetime string '{datetime_str}': {e}")
+        raise e
+
+
+# Test the function
+formatted_date = format_date("2024-03-06T05:00Z")
```

Comment:
We've got a few functions to format datetime. Might be good to collect the functions and put them under one class.

Reply:
@GregoryHo88 I made that into a backlogged issue